### DISCUSSION TITLE
Feedback: Burger Buff

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#708] [Ideas]
The burger seems odd being not as good as certain other similar foods. I understand why this is the case for balance reasons, but still. Rather than buffing its hunger or saturation values how about something unique like:

Please give the burger 30 seconds of Nutrition. That would make it not OP, but just different and would stand out from other foods, as it would be the only non-platter one that did.

**Comment by RaymondBlaze** [comment]
I assume you mean Nourishment by `Nutrition`? That effect is basically for foods from cooking pot. 30 seconds of Saturation is undoubtedly OP so I won't even take that in my guess.
Technically it's impossible to make every food unique enough, especially when you're using vanilla ingredient (meaning even Neapolitan style buffs are not applicable).
And I have to note that the burger and sandwiches are basically the easiest crafted foods in FD, they have no complicated ingredients and is crafted in crafting table (and even in shapeless recipes).
A possible boost for the burger and sandwiches would be making them faster to eat since they're "fast food".

**Reply by Aceplante** [reply]
No i dont think they should be eaten faster. That is exclusive to small foods which the burger is not. Why would 30 seconds be OP? its only 30 seconds...

**Reply by Aceplante** [reply]
Edit: apparently the beef burrito is cultural delights, not yours. That mod is just so well made and, like yours, fits the vanilla style perfectly, so I often get them confused. I still dont see why 30 seconds is that OP though. Maybe 10 then?

**Comment by vectorwing** [comment]
The foods have gone through a few iterations regarding stats and benefits over the years. Sandwiches used to have "burst effects" which lasted 10 seconds or so, making them good combat foods; but that turned out to be not only overpowered, but also imbalancing to other finger foods, as none of them were as good as sandwiches.

Currently, FD hard-defines a few rules for foods:
- Only **bowl meals** can have Comfort;
- Only **plate meals** can have Nourishment;
- Finger foods have no effects, but stack to 64 and need no containers.

This is partially a visual aid, and partially balance around acquiring the Cooking Pot. Though yes, the sandwiches and a handful of finger foods might be too expensive for their benefits currently, so it's worth looking into later.
